### PR TITLE
governance chapter

### PR DESCRIPTION
**Pull request by maelle**
cf #62

### CODE REVIEW

File: editor-meta.Rmd [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+* Blog editors are ["code owners"](https://github.com/ropensci/roweb2/blob/master/.github/CODEOWNERS) of `roweb2/content/technotes`, `roweb2/content/blog`, `roweb2/content/commcalls` according to [roweb2's CODEOWNERS file ](https://github.com/ropensci/roweb2/blob/master/.github/CODEOWNERS) so any PR touching a file under these paths will automatically request their review. The blog editor that will effectively handle the PR should unrequest the review from other blog editor(s).
+
+For adding a new blog editor, the team maintainer or any ropensci admin
+
+* ensures the user has [enabled 2FA](https://help.github.com/en/github/authenticating-to-github/securing-your-account-with-two-factor-authentication-2fa),
+
+* [adds them to the team](https://github.com/orgs/ropensci/teams/blog-editors/members) as Member (the default role).
+
+BLABLABLA what to add regarding access to this very repo i.e. blog-guidance?
```
Comment (maelle) [inline editor-meta.Rmd]: if roweb2 were under ropensci-org, easy, we'd add roweb2 to the team (that'd be called "ropensci-org/blog-editors).

as it's not, I suppose the blog editor should separately gain write access to this repo.

File: editor-meta.Rmd [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Posts authored by staff
```
Comment (maelle) [inline editor-meta.Rmd]: Maybe add something about staff members' being responsible for

* tweeting about their post from rOpenSci Twitter account (either direct access or Tweetdeck access)
* creating the discuss forum blabla

In both cases add a link to the relevant sections of the editor guide?

Comment (maelle) [reply editor-meta.Rmd]: Also, do staff members have to tweet at 8am Pacific or can they tweet at any time?

Comment (stefaniebutland) [reply editor-meta.Rmd]: I will add this before merging (I *think* I can commit, then merge. Can't recall if I tried before)

File: editor-meta.Rmd (hunk 1 above)
Comment (stefaniebutland) [reply editor-meta.Rmd]: @maelle I'm not sure what is needed here. Can you please add?

File: editor-meta.Rmd [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* [adds them to the team](https://github.com/orgs/ropensci/teams/blog-editors/members) as Member (the default role).
+
+The blog editors can filter their notifications, e.g. using the [`reason:team-mention` filter](https://github.com/notifications/beta?query=reason%3Ateam-mention).
+
+BLABLABLA what to add regarding access to this very repo i.e. blog-guidance?
+
+## Posts authored by staff
+
+* Staff members can bypass review for tech notes but they can choose to request a review by blog editors.
+* Staff members' blog posts should be scheduled together with Stefanie Butland.
```
Comment (stefaniebutland) [inline editor-meta.Rmd]:
```suggestion
* Staff members' blog posts should be scheduled together with Stefanie Butland.
* Staff members are responsible for creating the [discussion forum entry](#publish) for their own post and must [tweet from the rOpenSci Twitter account](#tweetpromo) (directly or via the Tweetdeck rOpenSci team) before tweeting from their personal account.
```

File: editor-meta.Rmd (hunk 2 above)
Comment (stefaniebutland) [reply editor-meta.Rmd]: committed

File: editor-meta.Rmd (hunk 1 above)
Comment (maelle) [reply editor-meta.Rmd]: I added something but not sure that's the sentence you wanted?

**Pull request merged**